examples/blueprint.yaml

```diff
@@ -25,9 +25,22 @@ node_templates:
               type: cloudify.vcloud.server_connected_to_port
             - target: example_port2
               type: cloudify.vcloud.server_connected_to_port
-            - target: manager_floating_ip
+            - target: example_floating_ip
               type: cloudify.vcloud.server_connected_to_floating_ip
 
+    example_server2:
+        type: cloudify.vcloud.nodes.Server
+        properties:
+            server:
+                name: cfy-plugin-example-server
+                catalog: giga
+                template: plugin-test-VApp
+            management_network: CFY-Internal
+            vcloud_config: { get_property: [vcloud_configuration, vcloud_config] }
+        relationships:
+            - target: example_public_nat_for_vm
+              type: cloudify.vcloud.public_nat_connected_to_vm
```

Comment:
If you specifying this relationship in server properties, I think it make sense to name it 'server_connected_to_public_nat' to be consistent with direction the relationship (from server to public nat).